go/vt/mysqlctl/azblobbackupstorage/azblob.go

```diff
@@ -36,26 +37,44 @@ import (
 
 var (
 	// This is the account name
-	accountName = flag.String("azblob_backup_account_name", "", "Azure Account Name for Backups")
+	accountName = flag.String("azblob_backup_account_name", "", "Azure Account Name for Backups, alternative environment paramater is VITESS_AZBLOB_ACCOUNT_NAME")
 
 	// This is the private access key
-	accountKey = flag.String("azblob_backup_account_key", "", "Azure Account Key")
+	accountKey = flag.String("azblob_backup_account_key", "", "Azure Account Key, alternative environment paramater is VITESS_AZBLOB_ACCOUNT_KEY")
```

Comment:
The env var is fine, but we shouldn't offer a flag at all unless it's a file path. It's too easy to leak passwords/keys if they're baked into command lines, so we shouldn't encourage bad practices.

```suggestion
	accountKey = flag.String("azblob_backup_account_key_file", "", "path to a file containing the Azure Storage account key; if this flag is unset, the environment variable VITESS_AZBLOB_ACCOUNT_KEY will be used as the actual key value (not a file path)")
```